src/pages/SupAdDashboard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Sidebar from '../components/Sidebar';
+import Chart from '../components/Chart';
+import Card from '../components/Card';
+
+const SupAdDashboard = () => {
+  return (
+    <div className="flex flex-col bg-light-bg dark:bg-dark-frame-bg">
+      <div className="flex flex-row">
+        <Sidebar style="hidden lg:flex" />
```

Comment:
Since the **Sidebar** will be shared across all dashboards, it would be  better to add it to the routes. Though it not a deal-breaker it would be great